src/head.c

```diff
@@ -182,12 +182,21 @@ void head_scaled_dimensions(struct Head *head) {
 		return;
 	}
 
-	if (head->transform % 2 == 0) {
+	if (head->transform == WL_OUTPUT_TRANSFORM_NORMAL || 
```

Comment:
`head->transform % 2 == 0` is actually enough, as multiples of 90 are odd values.